### DISCUSSION TITLE
Can't get my HASPone to respond to any presses on the touchscreen

### DISCUSSION THREAD

**Discussion by sanity** [HASwitchPlate/HASPone#214] [Q&A] [answered]
I've got the device set up through Home Assistant and am able to add things like the weather to it through blueprints.

However, no matter what I do I can't get it to respond to pushing any of the screen buttons. This is true of the page buttons at the bottom, and also a toggle button blueprint I created. 

I've used telnet to look at the device's logs but don't see anything when I press anything on the screen.

I'm starting to worry that my touchscreen isn't working or perhaps was never a touchscreen to begin with (I purchased it pre-built on etsy).

Does anyone know what I might be missing?

```
MQTT Status: Connected
MQTT ClientID: kitchenplate1-a848faff5a43
HASPone FW Version: 1.05
LCD Model: NX3224F024_011R
LCD FW Version: 3
LCD Active Page: 1
LCD Serial Speed: 921600
CPU Frequency: 160MHz
Sketch Size: 661728 bytes
Free Sketch Space: 2482176 bytes
Heap Free: 18624
Heap Fragmentation: 3
ESP core version: 3.0.2
IP Address: 192.168.1.229
Signal Strength: -69
Uptime: 10
Last reset: Software/System restart
```

**Comment by aderusha** [comment, marked as answer]
Hey man, if you are connecting via telnet and nothing is happening when you press the screen, something for sure is wrong :|

Given that this is a pre-built unit you purchased, the first thing we can do is to simply return the unit and I'll send a new one out.  I don't want to have to waste your time troubleshooting what should have been a working unit on day 1.  So, shoot me an email at allen@derusha.org if you'd like to arrange that.

Alternately, if you'd like to dig into this, the first step would be to remove the 4 #1 phillips screws from the rear of the enclosure (unplugged first!), then to remove the 2x 1.5mm hex head screws holding the PCB into the case.  This will let you remove the entire unit and power it via USB for testing.  Sometimes the plastic of the enclosure can get in the way of things preventing touch from working.  Everything is powered up and tested for touch response prior to shipment, but USPS does what they do and maybe it's just sitting in there wrong?

**Reply by sanity** [reply]
Thank you, I'll try a bit of local testing first before returning it, will email. Thank you!
